### ISSUE TITLE
Inline DatePicker is not working after close/open

### ISSUE THREAD

**Issue by nikolatasikj** [NG-ZORRO/ng-zorro-antd#6897]
### Reproduction link
[https://stackblitz.com/edit/angular-ivy-eo5cg9](https://stackblitz.com/edit/angular-ivy-eo5cg9)

### Steps to reproduce
1. Click on 'Select a date' input to open the dropdown
2. Click outside to close the dropdown (you can also go through radio buttons before closing if you want)
3. Click on 'Select a date' input to open the dropdown again

### What is expected?
When you close and open the dropdown again, all inline DatePickers should be there

### What is actually happening?
When you open dropdown for the first time all DatePickers are there, but when you close it once and open it again, all DatePickers disappear, they are in DOM (opacity 0 is added on ant-picker-wrapper class), but not rendered properly. 
You can reproduce the same in your documentation example for inline DatePickers -> https://ng.ant.design/components/date-picker/en#:~:text=Default-,Week,-Month
If you click on "Week" and then click "Default" DatePicker will disappear.

| Environment | Info |
|---|---|
| ng-zorro-antd | 12.0.1 |
| Browser | Google Chrome - Version 92.0.4515.107  |

---
Just to mention that Angular Ivy is disabled on the reproduction link because ngcc can't compile all your modules and it throws error, can you please also fix this. You can go in tsconfig.json and under angularCompilerOptions > enableIvy set true
This is happening on Angular 12, I think that on Angular 11 everything works as it should.

**Comment by wenqi73** [comment]
@nikolatasikj Thank you for reporting. It's caused by `slideMotion` animation, need to move the element into overlay template:
```html
<div
  class="ant-picker-wrapper"
  [nzNoAnimation]="!!noAnimation?.nzNoAnimation"
  [@slideMotion]="'enter'"
  style="position: relative;"
>
...
</div>
```

**Comment by nikolatasikj** [comment]
> @nikolatasikj Thank you for reporting. It's caused by `slideMotion` animation, need to move the element into overlay template:
> 
> ```
> <div
>   class="ant-picker-wrapper"
>   [nzNoAnimation]="!!noAnimation?.nzNoAnimation"
>   [@slideMotion]="'enter'"
>   style="position: relative;"
> >
> ...
> </div>
> ```

Yes, I mention that in PR https://github.com/NG-ZORRO/ng-zorro-antd/pull/6910

**Comment by LarsG** [comment]
Is there any recommended workaround for this Date Picker inline bug? The problem still occures in v12.0.1.

**Comment by nikolatasikj** [comment]
> Is there any recommended workaround for this Date Picker inline bug? At least as temporary solution until its fixed in new version, because the problem still occures in v12.0.1.

It's still not deployed, I guess it will be fixed in v12.0.2. For now you can try to find and override that `opacity: 0` that is caused by `@slideMotion` animation

**Comment by LarsG** [comment]
Our frontend dev came up with this as a temporary solution:

```
::ng-deep nz-date-picker>.ant-picker-wrapper {
  opacity  : 1 !important;
  transform: scaleY(1) !important;
}
```
